### DISCUSSION TITLE
how to create project board with team assignments?

### DISCUSSION THREAD

**Discussion by Alex-Jongman** [community/community#33600] [GitHub Education]
Hi,

by letting the students work on a GitHub classroom based project team assignment we also want to teach them SCRUM.
But I noticed that that giving the students admin permissions of the repository aren't sufficient any more to create a project board (like in de Project Classic situation).
They now would need permissions on organization level to create a GitHub projects board by them self. 
Is there a way to automatically generate GitHub project boards for each team that starts on a GitHub classroom assignment?

Alex

**Comment by mikecrabb** [comment]
+1 for this, having the same issue and cant find an easy solution
